eva-tools/src/main/java/embl/ebi/variation/eva/vcfdump/VariantExporterController.java

```diff
@@ -170,16 +184,15 @@ private void checkHeaders(Map<String, VCFHeader> headers) {
 
             Path outputPath = Paths.get(outputDir).resolve(studyId + suffix);
             outputFilePaths.put(studyId, outputPath.toString());
-            // TODO: does the get sequence dictionary method throw any exception?
-//            // get sequence dictionary from header
-//            SAMSequenceDictionary sequenceDictionary;
-//            try {
-//                sequenceDictionary = headers.get(studyId).getSequenceDictionary();
-//            } catch (Exception e) {
-//                // TODO: log error
-//                sequenceDictionary = null;
-//            }
-            SAMSequenceDictionary sequenceDictionary = headers.get(studyId).getSequenceDictionary();
+            // get sequence dictionary from header
+            SAMSequenceDictionary sequenceDictionary;
+            try {
+                sequenceDictionary = headers.get(studyId).getSequenceDictionary();
+            } catch (SAMException e) {
+                logger.warn("Incorrect sequence dictionary in study {}: {}", studyId, e.getMessage());
+                logger.warn("Sequence dictionary won't be included in study {} output VCF header", studyId);
```

Comment:
Not sure people will understand what a "sequence dictionary" means XD I would suggest something less implementation-specific like:

> Incorrect sequence / contig meta-data found in study {}: {}
> It won't be included in study {} output VCF header

This they will know for sure what it is.